src/NUnitFramework/nunitlite/OutputManager.cs

```diff
@@ -120,7 +120,7 @@ public void WriteTestFile(ITest test, OutputSpecification spec)
                         "spec");
             }
 
-            outputWriter.WriteTestFile(test, outputPath);
+            outputWriter.WriteTestFile(test, filter, outputPath);
             Console.WriteLine("Tests ({0}) saved as {1}", spec.Format, outputPath);
         }
     }
```

Comment:
@CharliePoole  I have now removed the filtering that happens in the output writer